src/USER-CAC/pair_CAC_sw.h

```diff
@@ -0,0 +1,118 @@
+/* -*- c++ -*- ----------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   http://lammps.sandia.gov, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+#ifdef PAIR_CLASS
+
+PairStyle(CAC/sw,PairCACSW)
+
+#else
+
+#ifndef LMP_PAIR_SW_CAC_H
+#define LMP_PAIR_SW_CAC_H
+
+//#include "asa_user.h"
+#include "pair.h"
+#include "pair_CAC.h"
+#include <stdint.h>
+
+
+namespace LAMMPS_NS {
+
+class PairCACSW : public PairCAC {
+ public:
+  PairCACSW(class LAMMPS *);
+  virtual ~PairCACSW();
+
+  void coeff(int, char **);
+  virtual void init_style();
+  virtual double init_one(int, int);
+ // virtual double single(int, int, int, int, double, double, double, double &);
+  //double LJEOS(int);
+  /*
+  struct asa_objective_struct;
+  typedef asa_objective_struct asa_objective;
+
+  struct asacg_parm_struct;
+  typedef asacg_parm_struct asacg_parm;
+
+  struct asa_parm_struct;
+  typedef asa_parm_struct asa_parm;
+  */
+	struct Param {
```

Comment:
please reformat all source files (where possible) to match LAMMPS programming style (this file just reminded me):
- indent depth 2 characters
- no tabs
- no trailing whitespace
- remove excessive empty lines